src/daemon/config.h

```diff
@@ -0,0 +1,25 @@
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef INTERFACECONFIG_H
```

Comment:
This file should be called 'interfaceconfig.h' or here we should replace 'interfaceConfig' with 'Config'

Reply:
What do you think of the name InterfaceConfig? It seems right, but I wanted to check. If InterfaceConfig is good, I'll rename the file.